anyway/request_params.py

```diff
@@ -154,6 +164,32 @@ def extract_road_segment_location(road_segment_id):
     return {"name": "location", "data": data, "gps": gps, "text": text}
 
 
+# todo: fill both codes and names into location
+def extract_street_location(input_vals: dict):
+    vals = fill_missing_street_values(input_vals)
+    # noinspection PyDictCreation
+    data = {"resolution": BE_CONST.ResolutionCategories.STREET}
+    for k in ["yishuv_name", "yishuv_symbol", "street1", "street1_hebrew"]:
+        data[k] = vals[k]
+    text = get_street_location_text(vals["yishuv_name"], vals["street1_hebrew"])
+    # fake gps - todo: fix
+    gps = {"lat": 32.825610, "lon": 35.165395}
```

Comment:
Is that intended?

Reply:
Yes, until we will get accurate coordinates in our input.